openage/convert/main.py

```diff
@@ -174,10 +180,11 @@ def acquire_conversion_source_dir():
         print("found environment variable AGE2DIR")
     else:
         # TODO: use some sort of GUI for this (GTK, QtQuick, zenity?)
-        proposals = [proposal for proposal in _get_source_dir_proposals()
-                     if os.path.exists(_expand_relative_path(proposal))]
+        proposals = set(proposal for proposal in _get_source_dir_proposals()
+                        if os.path.exists(_expand_relative_path(proposal)))
         print("Select an Age of Kings installation directory. "
               "Insert the index of one of the proposals, or any path:")
+        proposals = sorted(proposals)
```

Comment:
A loop around everything sounds useful, will do. Everything else… do it if you wish, for me it seemed most convenient to "try if it's a valid index, otherwise try as path".
However, I just note I didn't quite implement that ;) will do